### DISCUSSION TITLE
Where is the CSF catalog?

### DISCUSSION THREAD

**Discussion by aj-stein-gsa** [usnistgov/OSCAL#2051] [Q&A] [answered]
I can see https://github.com/usnistgov/OSCAL/issues/1949 is now marked completed. Is the resulting CSF 1.1 catalog in OSCAL publicly available now?

**Comment by iMichaela** [comment, marked as answer]
@aj-stein-gsa -- CSF v2.0 is done (the tool generating it is [here](https://github.com/usnistgov/capordino/blob/main/src/main/java/gov/nist/capordino/cprt/conversion/AbstractOscalConverter.java)).  CSF team wanted initially the manage the OSCAL CSF v2 rom circa, but since I don't see it happening, I will release it with the next oscal-content release. Do you need it sooner?

**Reply by aj-stein-gsa** [reply]
No, I do not need it. It was out of personal interest, just curious. I will look at the tool later.

**Reply by iMichaela** [reply]
So this is satisfactory for now? We have CSF v1.1 in OSCAL but the CPRT data includes mappings to 800-53 controls as references and the discussion with CSF team indicated an OSCAL version of CSF v1.1 is not needed.
